client/public/locales/en/translation.json

```diff
@@ -14,6 +14,7 @@
         "cancel": "Cancel",
         "checkDocumentation": "Check documentation",
         "clearAllFilters": "Clear all filters",
+        "clearRepositoryNotSupported": "This action is disabled when RWX option is not supported",
```

Comment:
We should get this text vetted by @jortel  / @rromannissen

Reply:
@ibolton336, let me run ask them.

Reply:
Don't have much of a preference but would omit the word `option`.

Reply:
Also, you'll need to gray/disable the size field as well when RWX not supported.

Reply:
What about "This action is disabled when RWX volumes are not available"?

Reply:
@rromannissen, we're going to follow-up in separate PR as this one got merged in order to have the feature checked-in ASAP. Thanks